sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_patch.py

```diff
@@ -1317,8 +649,11 @@ def _convert_from_generated_model(generated_model):
             ),
         )
 
+    # pylint: disable=arguments-renamed
     @distributed_trace
-    def get_document_status(self, translation_id: str, document_id: str, **kwargs: Any) -> DocumentStatus:
+    def get_document_status(  # type: ignore
```

Comment:
Does the generated client expose this method? And if so, do we still need to patch it here?

Reply:
Ah I see...yes that is an unfortunate limitation we've been dealing with, but since I think it's more noise to patch in a client library instead of just using the `@clientName` decorator, I'd say just use the decorator over the typespec property inline in the code. And make sure to specify Python if it's the only language that renamed it.

Reply:
Added the decorator in routes.tsp on the id parameter and regenerated the SDK and cleaned up the patching for those operations.